vividus-plugin-aws-s3/src/test/java/org/vividus/aws/s3/steps/S3BucketStepsTests.java

```diff
@@ -64,22 +68,37 @@
     private S3BucketSteps steps;
 
     @Test
-    public void uploadFileTest() throws Exception
+    public void uploadResourceTest() throws Exception
     {
-        String contentType = "contentType";
-        byte[] resource = "string".getBytes(StandardCharsets.UTF_8);
-
         ObjectMetadata objectMetadata = Mockito.mock(ObjectMetadata.class);
         ByteArrayInputStream inputStream = Mockito.mock(ByteArrayInputStream.class);
         PowerMockito.mockStatic(ResourceUtils.class);
-        PowerMockito.when(ResourceUtils.loadResourceAsByteArray(CSV_FILE_PATH)).thenReturn(resource);
+        PowerMockito.when(ResourceUtils.loadResourceAsByteArray(CSV_FILE_PATH)).thenReturn(TEST_FILE_CONTENT);
```

Comment:
The merge was done in a wrong way, PowerMock usage was eliminated: https://github.com/vividus-framework/vividus/commit/e7e5a1fb66ad1d02faf42e152e1adc5d9955ffde